src/data/resolvers/queries/knowledgebase.js

```diff
@@ -0,0 +1,120 @@
+import { KbTopics, KbCategories, KbArticles, Users } from '../../../db/models';
+
+export default {
+  kbTopic(root, { topicId }) {
+    return KbTopics.findOne({ _id: topicId }).then(topic => ({
```

Comment:
Promises chain should be more readable. https://javascript.info/promise-chaining

Reply:
okay